inc/image.php

```diff
@@ -52,6 +52,13 @@ class Optml_Image {
 	 */
 	private $source_url = '';
 
+	/**
+	 * Cache Buster passed from  Optimole Service.
+	 *
+	 * @var string Active Cache Buster
+	 */
+	private $cache_buster = '1';
```

Comment:
we can use cache_buster value on user who cleared cache, for rest of them use the path token.